values.yaml

```diff
@@ -32,6 +32,8 @@ srv:
     tag: latest
   env:
     SPRING_PROFILES_ACTIVE: cloud
+    BP_JVM_VERSION: 17
+    BP_JVM_TYPE: JRE
```

Comment:
Is this file relevant during container build time? I would have imagined that the decision which JRE to use is already made when building the containers with `pack`. Does this take these environment variables into account?

Reply:
Hah, you right. It can detect the required Java even without hints. Cool.